examples/min-platform/embedding/Cargo.toml

```diff
@@ -14,7 +14,7 @@ anyhow = { workspace = true }
 # Note that default-features of wasmtime are disabled and only those required
 # are enabled, in this case compilation is done in the guest from the wasm text
 # format so `cranelift` and `wat` are enabled.
-wasmtime = { workspace = true, features = ['runtime'] }
+wasmtime = { workspace = true, features = ['runtime', 'async'] }
```

Comment:
I think it's ok to leave this out for now and mostly just ensure this is `cargo check`'d somewhere in CI (to avoid someone copy/pasting this `Cargo.toml` example and forgetting to turn this back off)

Reply:
Done.